app/src/main/java/io/homeassistant/companion/android/sensors/NextAlarmManager.kt

```diff
@@ -0,0 +1,88 @@
+package io.homeassistant.companion.android.sensors
+
+import android.app.AlarmManager
+import android.content.Context
+import android.util.Log
+import io.homeassistant.companion.android.domain.integration.Sensor
+import io.homeassistant.companion.android.domain.integration.SensorRegistration
+import java.text.SimpleDateFormat
+import java.util.Calendar
+import java.util.Date
+import java.util.GregorianCalendar
+import java.util.TimeZone
+
+class NextAlarmManager : SensorManager {
+    companion object {
+
+        private const val TAG = "NextAlarm"
+        var triggerTime = 0L
+        var local = ""
+        var utc = ""

```

Comment:
These should be declared in the method they are used since they are only used there.  The constant should stay.